test/convnets.jl

```diff
@@ -18,57 +16,95 @@ GC.gc()
             @test_throws ArgumentError VGG(sz, batchnorm = bn, pretrain = true)
         end
         @test gradtest(m, x_224)
-        GC.safepoint()
-        GC.gc()
+        _gc()
     end
 end
 
-GC.safepoint()
-GC.gc()
-
 @testset "ResNet" begin
-    @testset "ResNet($sz)" for sz in [18, 34, 50, 101, 152]
-        m = ResNet(sz)
-        @test size(m(x_256)) == (1000, 1)
-        if (ResNet, sz) in PRETRAINED_MODELS
-            @test acctest(ResNet(sz, pretrain = true))
-        else
-            @test_throws ArgumentError ResNet(sz, pretrain = true)
-        end
-        @test gradtest(m, x_256)
-        GC.safepoint()
-        GC.gc()
-    end
+    # Tests for pretrained ResNets
+    ## TODO: find a way to port pretrained models to the new ResNet API
+    # @testset "ResNet($sz)" for sz in [18, 34, 50, 101, 152]
+        # if (ResNet, sz) in PRETRAINED_MODELS
+        #     @test acctest(ResNet(sz, pretrain = true))
+        # else
+        #     @test_throws ArgumentError ResNet(sz, pretrain = true)
+        # end
+    # end
```

Comment:
Leaving note to remember that we can't tag before this is resolved.

Reply:
I have yet to work seriously on this 😅 But I think it should be okay from previous discussions with @darsnack. Will try and work something out next week maybe

Reply:
Oh you don't have to worry about it now, just needed a record in case we got a request to tag in the interim.

Reply:
Because those are already being handled below 😅 It's the same models, just without the `ResNet` wrapper

Reply:
Not quite though cause `ResNet` doesn't have any drop paths, etc. And I think if you only want to test something once, it should be via calling `ResNet` (the highest level API) to ensure that these methods never break or error accidentally on a change.